dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/importer/EventManager.java

```diff
@@ -429,4 +448,34 @@ else if ( CREATE == importStrategy )
 
         return failedEvents;
     }
+
+    /**
+     * Validates the events using the supplied list of validators.
+     * <p>
+     * Only returns the ImportSummary for Events that *did* not pass validation
+     *
+     */
+    private List<ImportSummary> run( WorkContext workContext, List<Event> events, List<? extends Checker> checkers )
```

Comment:
I wonder why this `run` method became needed. How would this work before? Was this in one of the deleted files?

Reply:
Inlined the code as-is from

https://github.com/dhis2/dhis2-core/blob/f8054738d719c2d5251de87bf7d55108db2c13cd/dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/importer/EventImporterValidationRunner.java#L55